### PR TITLE
fix(test): upgrade account tests to Kotlin and fix

### PR DESCRIPTION
**Pull request by djansen-redhat**
JIRA issue URL: n/a

Convert the account tests to Kotlin, add to suite and repair where necessary

## Checklist

- JIRA link
- Check target branch
- Make sure all commit statuses are green or otherwise documented reasons to ignore
- QA needs to evaluate against the JIRA ticket
- Changed files and commits make sense for this PR

See [Zanata Development Guidelines](https://github.com/zanata/zanata-platform/wiki/Development-Guidelines) more for information.

----
*This template can be updated in .github/PULL_REQUEST_TEMPLATE.md*

### CODE REVIEW

**Comment by seanf** [comment]
@djansen-redhat did the allFuncTests thing work?

File: server/zanata-war/src/main/webapp/WEB-INF/layout/language/settings-tab.xhtml [hunk 1]
```diff
@@ -104,8 +104,12 @@
...
 
         <div class="l--push-top-1">
-          <a4j:commandLink action="#{languageAction.saveSettings()}" styleClass="loader button button--primary"
-            render="language-info" onbegin="jQuery(this).attr('disabled', 'disabled')" oncomplete="jQuery(this).removeAttr('disabled').removeClass('is-active')">
+          <a4j:commandLink id="save-language-settings"
+                           action="#{languageAction.saveSettings()}"
+                           styleClass="loader button button--primary"
+                           render="language-info"
+                           onbegin="jQuery(this).attr('disabled', 'disabled')"
+                           oncomplete="jQuery(this).removeAttr('disabled').removeClass('is-active')">
```
Comment (seanf) [inline server/zanata-war/src/main/webapp/WEB-INF/layout/language/settings-tab.xhtml]: I'm glad you kept the order the same!

But this indentation isn't standard. I think we indent continuations by 2 spaces in XML.

**Comment by djansen-redhat** [comment]
@seanf unfortunately no

File: server/zanata-war/src/main/webapp/WEB-INF/layout/language/settings-tab.xhtml (hunk 1 above)
Comment (djansen-redhat) [reply server/zanata-war/src/main/webapp/WEB-INF/layout/language/settings-tab.xhtml]: Oops, fixed

**Pull request merged**